Jenkinsfile

```diff
@@ -3,12 +3,11 @@ import org.stan.Utils
 
 def utils = new org.stan.Utils()
 
-def setupCXX(failOnError = true) {
+def setupCXX(failOnError = true, CXX = env.GCC) {
```

Comment:
I think it makes more sense to have the default be env.CXX because it's used more that way